1.0/en/0x10-C04-Infrastructure.md

```diff
@@ -2,23 +2,23 @@
 
 ## Control Objective
 
-AI infrastructure must be hardened against privilege escalation, supply chain tampering, and lateral movement through secure configuration, runtime isolation, trusted deployment pipelines, and comprehensive monitoring. Only authorized, validated infrastructure components and configurations reach production through controlled processes that maintain security, integrity, and auditability.
+AI infrastructure must be hardened against privilege escalation, supply chain tampering, and lateral movement through secure configuration, runtime isolation, trusted deployment pipelines, and comprehensive monitoring. Only validated and authorized infrastructure components reach production through controlled processes that ensure security, integrity, and auditability.
 
 **Core Security Goal:** Only cryptographically signed, vulnerability-scanned infrastructure components reach production through automated validation pipelines that enforce security policies and maintain immutable audit trails.
 
 ---
 
 ## C4.1 Runtime Environment Isolation
 
-Prevent container escapes and privilege escalation through kernel-level isolation primitives and mandatory access controls.
+Prevent container escapes and privilege escalation through OS-level isolation primitives.
 
 | # | Description | Level | Role |
 |:--------:|--------------------------------------------------------------------------------------------|:---:|:---:|
-| **4.1.1** | **Verify that** all AI containers drop ALL Linux capabilities except CAP_SETUID, CAP_SETGID, and explicitly required capabilities documented in security baselines. | 1 | D/V |
-| **4.1.2** | **Verify that** seccomp profiles block all syscalls except those in pre-approved allowlists, with violations terminating the container and generating security alerts. | 1 | D/V |
-| **4.1.3** | **Verify that** AI workloads run with read-only root filesystems, tmpfs for temporary data, and named volumes for persistent data with noexec mount options enforced. | 2 | D/V |
-| **4.1.4** | **Verify that** eBPF-based runtime monitoring (Falco, Tetragon, or equivalent) detects privilege escalation attempts and automatically kills offending processes within organizational response time requirements. | 2 | D/V |
-| **4.1.5** | **Verify that** high-risk AI workloads execute in hardware-isolated environments (Intel TXT, AMD SVM, or dedicated bare-metal nodes) with attestation verification. | 3 | D/V |
+| **4.1.1** | **Verify that** all AI containers drop all Linux capabilities and add only explicitly justified capabilities. | 1 | D/V |
```

Comment:
Would generalize this to ` -- all AI workloads run with minimal permissions needed on the operating system, by e.g. dropping unnecessary Linux capabilities in case of a container.`